tests/jest/aggregate-themes.test.js

```diff
@@ -12,7 +41,72 @@ describe('Theme Aggregator', () => {
     });
   });
 
+  describe('aggregateThemes', () => {
+    it('returns an empty array for an empty default theme and empty array scope theme', () => {
+      const options = { theme: '', scoped: [] };
+
+      const files = ThemeAggregator.aggregateThemes(options);
+      expect(files).toEqual([]);
+    });
+
+    it('returns an empty array for an null default theme and null array scope theme', () => {
+      const options = { theme: null, scoped: null };
+
+      const files = ThemeAggregator.aggregateThemes(options);
+      expect(files).toEqual([]);
+    });
+
+    it('returns an empty array for a null default theme and scope theme array containing null and a non existent theme.', () => {
+      const options = { theme: null, scoped: [null, 'non-existent'] };
+
+      const files = ThemeAggregator.aggregateThemes(options);
+      expect(files).toEqual([]);
+    });
+
+    it('returns an empty array for a non existent default theme and empty scope theme', () => {
+      const options = { theme: 'unknown-theme', scoped: '' };
+
+      const files = ThemeAggregator.aggregateThemes(options);
+      expect(files).toEqual([]);
+    });
+
+    it('returns the aggregated scoped theme for a non existent default theme and defined scope theme', () => {
+      const options = { theme: '', scoped: ['terra-mock-dark-theme'] };
+
+      const files = ThemeAggregator.aggregateThemes(options);
+      const expected = ['../tests/jest/fixtures/themes/terra-mock-dark-theme/scoped-theme.scss'];
+
+      expect(files).toEqual(expected);
+    });
+
+    it('returns the aggregated default theme for a defined default theme and non existent scope theme', () => {
+      const options = { theme: 'terra-mock-dark-theme', scoped: ['unkown-theme'] };
+
+      const files = ThemeAggregator.aggregateThemes(options);
+      const expected = ['../tests/jest/fixtures/themes/terra-mock-dark-theme/root-theme.scss'];
+
+      expect(files).toEqual(expected);
+    });
+
+    it('returns the aggregated default theme for a defined default theme and empty scope theme', () => {
+      const options = { theme: 'terra-mock-dark-theme', scoped: [] };
+
+      const files = ThemeAggregator.aggregateThemes(options);
+      const expected = ['../tests/jest/fixtures/themes/terra-mock-dark-theme/root-theme.scss'];
+
+      expect(files).toEqual(expected);
+    });
+  });
+
   describe('aggregateTheme', () => {
+    it('warns if a blank value is passed as a themename and returns null', () => {
+      const files = ThemeAggregator.aggregateTheme('');
+
+      // eslint-disable-next-line no-console
+      expect(console.warn).toHaveBeenCalled();
```

Comment:
We should reset this mock before this test. Right now it will be true due to: https://github.com/cerner/terra-toolkit/pull/337/files?file-filters%5B%5D=.js#diff-ef7737a0dfae559557f63f4133b8f85fR29

Reply:
https://github.com/cerner/terra-toolkit/pull/337/commits/1a8a45efa6bfa87dd20ff4dc38195c459beb3e9c